src/main/java/org/betonquest/betonquest/compatibility/protocollib/api/GlowAPI.java

```diff
@@ -0,0 +1,238 @@
+package org.betonquest.betonquest.compatibility.protocollib.api;
+
+import com.comphenix.protocol.PacketType;
+import com.comphenix.protocol.ProtocolLibrary;
+import com.comphenix.protocol.events.PacketContainer;
+import com.comphenix.protocol.utility.MinecraftReflection;
+import com.comphenix.protocol.wrappers.WrappedDataWatcher;
+import com.comphenix.protocol.wrappers.WrappedWatchableObject;
+import lombok.CustomLog;
+import net.citizensnpcs.api.CitizensAPI;
+import net.citizensnpcs.api.npc.NPC;
+import org.bukkit.ChatColor;
+import org.bukkit.OfflinePlayer;
+import org.bukkit.entity.Entity;
+import org.bukkit.entity.Player;
+
+import java.lang.reflect.InvocationTargetException;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.List;
+import java.util.concurrent.CompletableFuture;
+
+/**
+ * Glow API class
+ */
+@CustomLog
+public class GlowAPI {
+
+    /**
+     * Constructor of Glow API instance
```

Comment:
please write a more detailed description for this class for the java docs

Reply:
Well this my problem that I had. Because I dont really good at english I don't know how to describe it well 😂. But I'll try